.github/workflows/cron-staging.yml

```diff
@@ -2,6 +2,8 @@ name: Cron-staging
 on:
   schedule:
     - cron: "0 4 * * *"
+    pull_request:
+      branches: [ main, 'stable/*' ]
```

Comment:
I would prefer to leave this condition in permanently, and filter by files changed using the `paths` filter, so that these integration tests will run when you change service-related files.